ports/raspberrypi/fwk_keyboard/mpconfigboard.h

```diff
@@ -0,0 +1,30 @@
+#define MICROPY_HW_BOARD_NAME "Framework Laptop 16 Keyboard (CircuitPython)"
+#define MICROPY_HW_MCU_NAME "rp2040"
+
+//#define BOARD_FLASH_SIZE (FLASH_SIZE - 0x4000 - CIRCUITPY_INTERNAL_NVM_SIZE)
+
+// External flash W25Q80DV
+#define EXTERNAL_FLASH_QSPI_DUAL
+#if QSPI_FLASH_FILESYSTEM
+//#define MICROPY_QSPI_DATA0                NRF_GPIO_PIN_MAP(1, 00)
+//#define MICROPY_QSPI_DATA1                NRF_GPIO_PIN_MAP(0, 21)
+//#define MICROPY_QSPI_DATA2                NRF_GPIO_PIN_MAP(0, 22)
+//#define MICROPY_QSPI_DATA3                NRF_GPIO_PIN_MAP(0, 23)
+//#define MICROPY_QSPI_SCK                  NRF_GPIO_PIN_MAP(0, 19)
+//#define MICROPY_QSPI_CS                   NRF_GPIO_PIN_MAP(0, 20)
+#endif
+
+#define CIRCUITPY_DRIVE_LABEL "CIRCUITPY"
+#define CIRCUITPY_BOOT_COUNTER 1
+#define FLASH_SIZE                  (0x100000)
+#define FLASH_PAGE_SIZE             (0x4000)
+
+#define BOARD_HAS_CRYSTAL 1
+
```

Comment:
```suggestion
```
You can delete all this.

Reply:
Take a look at other existing RP2040 boards, such as KB2040 and QT Py RP2040.

Reply:
Initially I copied it from the QT Py RP2040. But the flash chip is different. So that config won't even run.
Once I change the flash chip description to the correct one, it runs but there's no folder.

Reply:
Do other boards show CIRCUITPY? (Just making sure there's nothing weird about your host system.)

Do take away the lines above; they are for an nRF board, and maybe some other things, not RP2040. Your `mpconfigboard.h` should look at a lot the Feather or QT Py RP2040.

You can try re-creating CIRCUITPY from the REPL:
```py
import storage
storage.erase_filesystem()
```
but I'm thinking there might be something else wrong.

Reply:
The boards are not meant for general purpose IO, so we don't need many of the circuitpy modules.
I removed a bunch of them to get the fw size below 512K. Now there's plenty of space for the CIRCUITPY drive and it does work!

Reply:
Can I also limit the maximum size of the CIRCUITPY drive? It doesn't seem quite as straight forward. And the code comments say it expects at least 2M of flash to work.
My reason is that we put the serial number in the last sector of flash 1M-4096B. We'll lock this sector but I'm afraid the circuitpy drive might not handle this properly.

Reply:
We do this for the Teensy 4.x as well. I think we use a linker script setting there. On RP2040 we'll likely want a compile time option because RP2040 CP detects flash size automatically.

If you need a tester, I'm very interested in a Framework laptop. :-)